<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>SearchMedia</class>
 <widget class="QMainWindow" name="SearchMedia">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1144</width>
    <height>679</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QGridLayout" name="gridLayout">
    <item row="0" column="0">
     <layout class="QHBoxLayout" name="horizontalLayout">
      <item>
       <layout class="QVBoxLayout" name="viewMediaLayout">
        <item>
         <widget class="QTreeWidget" name="viewSearchTree">
          <property name="columnCount">
           <number>5</number>
          </property>
          <attribute name="headerCascadingSectionResizes">
           <bool>true</bool>
          </attribute>
          <attribute name="headerDefaultSectionSize">
           <number>128</number>
          </attribute>
          <column>
           <property name="text">
            <string>Афиша</string>
           </property>
          </column>
          <column>
           <property name="text">
            <string>Название</string>
           </property>
          </column>
          <column>
           <property name="text">
            <string>Год</string>
           </property>
          </column>
          <column>
           <property name="text">
            <string>Жанр</string>
           </property>
          </column>
          <column>
           <property name="text">
            <string>Тип</string>
           </property>
          </column>
         </widget>
        </item>
       </layout>
      </item>
      <item>
       <layout class="QVBoxLayout" name="controlMediaLayout">
        <item>
         <layout class="QHBoxLayout" name="typeSearchLayout">
          <item>
           <widget class="QLabel" name="label">
            <property name="text">
             <string>Тип:</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QComboBox" name="typeMedia">
            <item>
             <property name="text">
              <string>Фильмы+Сериалы</string>
             </property>
            </item>
            <item>
             <property name="text">
              <string>Фильмы</string>
             </property>
            </item>
            <item>
             <property name="text">
              <string>Сериалы</string>
             </property>
            </item>
           </widget>
          </item>
         </layout>
        </item>
        <item>
         <layout class="QHBoxLayout" name="mediaSourceLayout">
          <item>
           <widget class="QLabel" name="labelMediaSource">
            <property name="text">
             <string>Источник:</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QComboBox" name="mediaSource">
            <item>
             <property name="text">
              <string>TMDB</string>
             </property>
            </item>
           </widget>
          </item>
         </layout>
        </item>
        <item>
         <layout class="QHBoxLayout" name="langSearchLayout">
          <item>
           <widget class="QLabel" name="langSearchLabel">
            <property name="text">
             <string>Язык:</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QComboBox" name="langSearchComboBox">
            <item>
             <property name="text">
              <string>Английский</string>
             </property>
            </item>
            <item>
             <property name="text">
              <string>Русский</string>
             </property>
            </item>
           </widget>
          </item>
         </layout>
        </item>
        <item>
         <layout class="QHBoxLayout" name="horizontalLayout_3">
          <item>
           <widget class="QLabel" name="labelSearchWord">
            <property name="text">
             <string>Искомое:</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QLineEdit" name="searchWordEdit"/>
          </item>
         </layout>
        </item>
        <item>
         <widget class="QPushButton" name="searchButton">
          <property name="text">
           <string>Искать</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QPlainTextEdit" name="overviewMedia">
          <property name="readOnly">
           <bool>true</bool>
          </property>
          <property name="plainText">
           <string/>
          </property>
         </widget>
        </item>
        <item>
         <spacer name="verticalSpacer">
          <property name="orientation">
           <enum>Qt::Orientation::Vertical</enum>
          </property>
          <property name="sizeHint" stdset="0">
           <size>
            <width>20</width>
            <height>40</height>
           </size>
          </property>
         </spacer>
        </item>
        <item>
         <widget class="QProgressBar" name="progressSearch">
          <property name="value">
           <number>0</number>
          </property>
          <property name="textVisible">
           <bool>false</bool>
          </property>
          <property name="orientation">
           <enum>Qt::Orientation::Horizontal</enum>
          </property>
          <property name="invertedAppearance">
           <bool>false</bool>
          </property>
          <property name="format">
           <string/>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QPushButton" name="selectFindMediaButton">
          <property name="enabled">
           <bool>false</bool>
          </property>
          <property name="text">
           <string>Выбрать</string>
          </property>
         </widget>
        </item>
       </layout>
      </item>
     </layout>
    </item>
   </layout>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
